cmd/lifecycle/analyzer.go

```diff
@@ -175,18 +176,18 @@ func (aa analyzeArgs) analyze() (platform.AnalyzedMetadata, error) {
 		img imgutil.Image
 		err error
 	)
-	if aa.imageName != "" {
+	if aa.previousImage != "" {
```

Comment:
I think we could remove this guard as `aa.previousImage` will never be blank (because `a.imageName` will never be blank). We didn't have this guard in 0.11.x.

Reply:
Good catch! See 33f945a